grpc-examples/pom.xml

```diff
@@ -20,7 +20,7 @@
         <dependency>
             <groupId>io.grpc</groupId>
             <artifactId>grpc-netty-shaded</artifactId>
-            <version>1.40.1</version>
+            <version>1.42.1</version>
```

Comment:
Should probably use same as `grpc/pom.xml`:

```
    <dependency>
      <groupId>io.grpc</groupId>
      <artifactId>grpc-netty-shaded</artifactId>
      <version>${grpc.version}</version>
    </dependency>
```

where `grpc.version` defined in main-level pom.xml (although for some reason older version?)